src/Storefront/Controller/ProductController.php

```diff
@@ -148,9 +148,12 @@ public function saveReview(string $productId, RequestDataBag $data, SalesChannel
     #[Route(path: '/product/{productId}/reviews', name: 'frontend.product.reviews', defaults: ['XmlHttpRequest' => true], methods: ['GET', 'POST'])]
     public function loadReviews(string $productId, Request $request, SalesChannelContext $context): Response
     {
-        $this->checkReviewsActive($context);
-
-        $reviews = $this->productReviewLoader->load($request, $context, $productId, $request->get('parentId'));
+        /** @deprecated tag:v6.8.0 Remove complete try catch block, the exception is then `ProductException::reviewNotActive` */
```

Comment:
we would like to make it more clear, what to do with the next major preparation. 
so instead describing what to remove, sometimes better, sometimes worse, we would like to use feature flag conditions more often. 

```
if (!Feature::isActive('v6.8.0.0')) {
    // old behaviour
} else {
   // new behaviour
}
```

so with this, it is crystal clear, what needs to be done in the future :slightly_smiling_face: would be nice, if you could adjust the places like that

Reply:
Aye :-)